package.json

```diff
@@ -297,19 +297,34 @@
     },
     "scripts": {
         "vscode:prepublish": "npm run compile",
+        "build": "rimraf dist && webpack --mode production",
         "compile": "tsc -p ./",
         "watch": "tsc -watch -p ./",
-        "postinstall": "node ./node_modules/vscode/bin/install"
+        "postinstall": "node ./node_modules/vscode/bin/install",
+        "lint": "concurrently -n \"_eslint_,prettier\" \"eslint --ext .ts,.tsx,.js .\" \"prettier --check .\"",
+        "pretty": "prettier --write .",
+        "test": "npm run test:unit && npm run test:integration",
+        "test:unit": "jest --config jest.config.js --coverage",
+        "test:vscode": "ts-node ./test/runTest.ts",
+        "test:che4z": "jest --config jest.int.config.js --coverage -i",
+        "test:integration": "npm run test:che4z && npm run test:vscode",
+        "package": "mkdirp vsce && vsce package --yarn -o vsce/bridge-for-git.vsix"
```

Comment:
We don't have the following:
- webpack, linting, vscode-test and offline packaging

Please remove lines: 304, 305, 308, 311
On line 310 remove `&& npm run test:vscode`
replace `che4z` with `endevor`